components/tekton/tasks/deploy-vse-sync-testsuite.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  annotations:
+    tekton.dev/categories: Testing
+    tekton.dev/displayName: vse-framework
+    tekton.dev/tags: 'golang, vse-sync-testsuite'
+  name: deploy-vse-sync-testsuite
+spec:
+  description: This task runs the vse-sync-testsuite image.
+  params:
+    - default: 'quay.io/redhat-partner-solutions/vse-sync-testsuite:latest'
+      description: Image for test suite
+      name: test-image
+      type: string
+    - default: vse-sync-testsuite-config
+      description: |
+        Name of the config-volume map to mount
+      name: config_volume
+      type: string
+    - description: |
+        The tag for the image you want to run
+      name: kubeconfig_secret
+      type: string
+    - default: /usr/th/kubeconfig/
+      description: |
+        Path that the kubeconfig will be mounted into
+      name: kubeconfig_secret_dest_path
+      type: string
+    - default: /workspace/reports/vse-sync-testsuite.json
+      description: |
+        Path of test results report
+      name: report_path
+      type: string
+    - default: 'docker.io/stedolan/jq:latest'
+      description: Image for jq
+      name: jq-image
+      type: string
+  steps:
+    - image: $(params.test-image)
+      name: run-ptp-suite
+      script: |
+        #!/bin/sh
+
+        ./tests.test \
+          --config=/usr/th/cfg/test_config.yaml \
+          --env=/usr/th/cfg/env.yaml \
+          --ginkgo.json-report="$(params.report_path)"
+
+        echo $?
+      volumeMounts:
+        - mountPath: /usr/th/cfg/
+          name: config-volume
+        - mountPath: $(params.kubeconfig_secret_dest_path)
+          name: kubeconfig-secret
+    - image: $(params.jq-image)
```

Comment:
I think this can be hardcoded

Reply:
I hit the docker rate limit so they may well want to put it somewhere else.

Reply:
[This](https://quay.io/repository/toolhippie/jq) one looks updated